scripts/qa_crowbarsetup.sh

```diff
@@ -2521,6 +2524,7 @@ function prepare_proposals()
     iscloudver 6plus && ptfchannel="PTF"
     for machine in $cmachines; do
         ssh $machine "zypper mr -p 90 $ptfchannel"
+        ssh $machine "echo 0 > /proc/sys/kernel/printk"
```

Comment:
sorry, just ignore it, I just pushed my working branch, I'll  rebase on top of the other PR when it got merged